wcfsetup/install/files/style/icon/vendor/regular.scss

```diff
@@ -0,0 +1,19 @@
+@import 'variables';
+
+@font-face {
+  font-family: 'Font Awesome 5 Free';
```

Comment:
Consider calling the font just `Font Awesome` for better backwards compatibility with code that expects the old name.